### ISSUE TITLE
Suggestion: add "edit scrobble" button for now scrobbling track

### ISSUE THREAD

**Issue by flickur** [RudeySH/lastfm-bulk-edit#15]
I know this is perhaps not directly related to bulk editing but would be a nice QoL feature and I couldn't think of a better place to suggest this. Would it be possible to add the "edit scrobble" button to the current playing track so you could edit it straight away without waiting for it to finish scrobbling? 

![image](https://user-images.githubusercontent.com/40724723/148943923-db1369e7-69ce-46bb-82b9-e53d185d0a4f.png)

**Comment by RudeySH** [comment]
The "Scrobbling now" track hasn't actually been scrobbled yet. At least not until you've played at least 50% of the track. And I even suspect some scrobble integrations like Spotify don't actually scrobble until you're 100% done playing the track.

You can't edit a scrobble that doesn't exist yet.

Although, I wonder if you can use the new "Automatic edit" feature for tracks that haven't been scrobbled yet...

**Comment by flickur** [comment]
Yeah the automatic edit feature is what got me thinking about this.

**Comment by RudeySH** [comment]
I added the option to edit scrobbles in the Recent Tracks section on January 24th, 2022 (v0.4.0, 6773a72e539622ed051927ab23ca2115c4cd6594), which includes the "Scrobbling now" track. If you edit the now scrobbling track, be sure to check "Automatic edit", because otherwise it only edits your existing scrobbles of the track, and the incoming scrobble won't be edited. If you try to edit a track that you haven't scrobbled before, my userscript won't let you continue at the moment.

Today, I realized the edit scrobbles option in Recent Tracks was disappearing after 20 seconds, because Last.fm is continuously refreshing the list and my script did not yet account for that. This bug has been fixed today (v0.5.1, f5bd0eaf7f6c09cdd9715fdd1b54144ab0a4ae57).
